src/app/components/server/StatusPill.tsx

```diff
@@ -84,7 +84,7 @@ export const getExposureStatus = (
 ): StatusPillType => {
   if (isScanResult(exposure)) {
     if (exposure.manually_resolved) {
-      return "removed";
+      return "fixed";
```

Comment:
Yes, that was something that has been [mentioned by QA](https://mozilla-hub.atlassian.net/browse/MNTOR-3201?focusedCommentId=897865) that was not the expected behavior:

> […] After the auto-removal process is complete, it will change to a description stating it was auto-removed, even if it was manually removed before. […]